packages/core/src/renderer/Renderer.ts

```diff
@@ -40,73 +38,8 @@ export interface RenderProps {
   texLabels: boolean;
   canvasSize: [number, number];
   pathResolver: PathResolver;
+  titleCache?: Map<string, SVGElement>;
```

Comment:
These are used in `InteractivityOverlay` (line 68) to create a map from paths to svg elements. Otherwise, we would nee to loop through every single svg element every update to attach click listeners.

Reply:
yep makes sense now! kinda wish we had more metadata to tell what shape type a string path has (e.g. `A.icon -> Circle`), that way we can handle mouseover properly, i.e. using `pointer-event: bounding-box` for `Text` and `Equation`. is there a better way to do this without more boilerplate?